values.yaml

```diff
@@ -111,29 +124,13 @@ zookeeper:
   ## ref: https://github.com/kubernetes/charts/tree/master/incubator/zookeeper
   enabled: true
 
-  # Number of ZooKeeper replicas
-  servers: 1
-
-  ## Configure Zookeeper resource requests and limits
-  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
-  resources:
-    limits:
-      memory: "256Mi"
-    requests:
-      memory: "128Mi"
-
-  antiAffinity: "soft"
-
-  ## The JVM heap size to allocate to Zookeeper
-  heap: "128m"
-
-  ## The amount of PV storage allocated to each Zookeeper pod in the statefulset
-  storage: "2Gi"
-
-  ## Specify a Zookeeper imagePullPolicy
-  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
-  #imagePullPolicy: "IfNotPresent"
-
   ## If the Zookeeper Chart is disabled a URL and port are required to connect
   url: ""
   port: 2181
+
+ca:
+  enabled: true
+  port: 8443
+  token: secret-token-to-prevent-mitm
```

Comment:
I don't know how this would work exactly, but I am wondering if we could make this value 'optional' and if not specified, generate a UUID or something like that? Does Helm provide the ability to generate a unique ID?